plugin/src/test/groovy/com/novoda/test/TestProject.groovy

```diff
@@ -27,13 +27,16 @@ ${project.additionalConfiguration}
     String toolsConfig = ''
 
     TestProject(Closure<String> template) {
+        def projectName = "${System.currentTimeMillis()}"
```

Comment:
These changes seem to be fine but I'm not sure if they are necessary. I would prefer to try with CI without these first and see what it says.

Reply:
Gradle 5 wasn't happy about not having a `settings.gradle` and we were having issues with the new memory managment (hence the gradle.properties)

Reply:
settings.gradle was always a requirement. Just not in tests. I'm afraid what you mention could be just for your local setup. Have you run it without settings.gradle in the CI?

Of course this can be Ok to bring in but at the same time it would be beneficial to double check with CI and see what it says.

Reply:
I would prefer to keep the `settings.gradle` empty, as we really don't need to name the projects. I anything they should have the same name of the folder they're generated in.

Reply:
@mr-archano Removed the contents of the `settings.gradle` however, it was using the name already generated and used for the test project